core/controllers/classifier_test.py

```diff
@@ -392,12 +392,24 @@ def test_error_on_no_training_job_for_retrieving_model(self):
         new_exp = self.save_new_default_exploration(
             new_exp_id, feconf.SYSTEM_COMMITTER_ID, title='New title')
 
-        change_list = [exp_domain.ExplorationChange({
-            'cmd': 'edit_state_property',
-            'state_name': new_exp.init_state_name,
-            'property_name': exp_domain.STATE_PROPERTY_INTERACTION_ID,
-            'new_value': 'NumericInput'
-        })]
+        change_list = [
+            exp_domain.ExplorationChange({
+                'cmd': 'edit_state_property',
+                'state_name': new_exp.init_state_name,
+                'property_name': exp_domain.STATE_PROPERTY_INTERACTION_ID,
+                'new_value': 'NumericInput'
+            }),
+            exp_domain.ExplorationChange({
+                'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+                'property_name':
+                    exp_domain.STATE_PROPERTY_INTERACTION_CUST_ARGS,
+                'state_name': new_exp.init_state_name,
+                'new_value': {
+                    'inputGreaterThanOrEqualToZero': {
```

Comment:
Could this be changed to "requireNonnegativeInput"? This follows how we name other boolean fields in interaction_specs.json.

Reply:
done